builtin/settingtypes.txt

```diff
@@ -499,6 +499,12 @@ water_wave_length (Waving liquids wavelength) float 20.0 0.1
 #    Requires: enable_waving_water
 water_wave_speed (Waving liquids wave speed) float 5.0
 
+#   When enabled, crude liquid reflections are simulated.
+#   When waving liquids are enabled, the wave parameters will affect these reflections.
+#
+#   Requires: enable_dynamic_shadows
+enable_water_reflections (Liquid reflections) bool false
```

Comment:
enable_water_reflections now shows twice in the settings. We should remove the old entry.